tekton/pipelines/source-container.yaml

```diff
@@ -38,3 +38,23 @@ spec:
           value: "$(params.OSBS_IMAGE)"
         - name: USER_PARAMS
           value: "$(params.USER_PARAMS)"
+  finally:
+    - name: source-container-exit
```

Comment:
the name of the build task above(https://github.com/containerbuildsystem/atomic-reactor/pull/1813/files#diff-37fd879ef818700016f8f18b08a45989f6a7eaa79c3090b68180c1d541a1402cR22) should also be consistent with this naming

Reply:
Update, PTAL.